app/decorators/hearing_decorator.rb

```diff
@@ -0,0 +1,39 @@
+# frozen_string_literal: true
+
+class HearingDecorator < BaseDecorator
+  def provider_list
+    return t('generic.not_available') if providers.blank? || providers.empty?
```

Comment:
I think you only need `providers.blank?` here as is the same as `.empty?` for arrays and active record relations.

Reply:
It caused errors without the blank? but that could just be the tests. likewise in the other places. I can revisit to check

Reply:
Does it work with `blank?` but not `empty?`?

Reply:
done